docs/pages/connect-your-client/introduction.mdx

```diff
@@ -0,0 +1,330 @@
+---
+title: Introduction to Teleport Clients
+description: The basics of connecting to resources with Teleport
+---
+
+This guide covers the basics of authenticating to Teleport and
+accessing resources. It's written for end-users of resources protected by
+Teleport, and includes links to more detailed documentation at the end.
+
+## Client tools
+
+### tsh
+
+`tsh` lets you authenticate to Teleport, and list and connect to resoruces. After [downloading](/downloads)
+and installing `tsh`, sign in to your Teleport cluster:
+
+<ScopedBlock scope={["oss"]}>
```

Comment:
Should we add a tab for SSO users to the `oss` `ScopedBlock`? I think OSS Teleport still supports authentication via GitHub.

Reply:
To address this and the comment below I'm removing the scoped blocks. We'll have just tabbed examples for local and SAML users, and use the Vars component to indicate both user and proxy values.

Reply:
Sounds good!